src/pages/AppDetail.svelte

```diff
@@ -9,9 +10,8 @@
     });
 </script>
 
-<h1>{params.app}</h1>
-
 {#if app}
+  <h1>{params.app}</h1>
```

Comment:
I'm pretty sure this code is out of date. Could you rebase/pull changes from the most recent version of the `main` branch?